spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/vault/SpringVaultClientConfiguration.java

```diff
@@ -125,6 +123,13 @@ public SslConfiguration sslConfiguration() {
 		return new SslConfiguration(keyStoreConfiguration, trustStoreConfiguration);
 	}
 
+	@Override
+	public RestOperations restOperations() {
```

Comment:
Note: The need to override this method should be considered a workaround. Can be removed once spring-projects/spring-vault#546 is resolved.

Reply:
Hi,
I need to implement it with namespaces. There is a better way to do it? 

`@Override
public RestOperations restOperations() {
    RestTemplate restTemplate = (RestTemplate) super.restOperations();
    // To define a namespace
    List<ClientHttpRequestInterceptor> clientHttpRequestInterceptorList = new LinkedList<>();
    ClientHttpRequestInterceptor namespaceInterceptor = VaultClients.createNamespaceInterceptor(NAMESPACE);
    clientHttpRequestInterceptorList.add(namespaceInterceptor);
    restTemplate.setInterceptors(clientHttpRequestInterceptorList);
    //

    return restTemplate;
}`